$NetBSD: patch-ad,v 1.3 2009/05/17 22:28:51 dholland Exp $

Remove bogus check that generates invalid shell syntax in the output.

Upstream: already fixed in upstream git.

--- configure.in.orig	2009-05-17 18:18:27.000000000 -0400
+++ configure.in	2009-05-17 18:18:38.000000000 -0400
@@ -358,7 +358,6 @@ linux)
 #include <net/if.h>
 #include <linux/types.h>
 	])
-	AC_CHECK_HEADERS()
 	AC_LBL_TPACKET_STATS
 	AC_LBL_LINUX_TPACKET_AUXDATA_TP_VLAN_TCI
 	;;
